openfisca_core/enumerations.py

```diff
@@ -6,6 +6,7 @@ def __init__(self, varlist, start = 0):
         self._vars = {}
         self._nums = {}
         self._count = 0
+        self.list = varlist
```

Comment:
If the user gives a value that is not in the list, would it be possible to catch it like this :

```py
class Enum(object):
    def __init__(self, varlist, start = 0):
        self._vars = {}
        self._nums = {}
        self._count = 0
        self.list = varlist
        for var in varlist:
            self._vars.update({self._count + start: var})
            self._nums.update({var: self._count + start})
            self._count += 1

    def __getitem__(self, var):
        try:
            return self._nums[var]
        except KeyError:
            valid_keys = ', '.join(self.list)
            log.error("' {} ' is not a valid entry for this variable. The available values are: {}".format(var, valid_keys))
            raise
```